client/src/pages/SignInModal.tsx

```diff
@@ -0,0 +1,112 @@
+import "./Modal.scss";
+import React, { FunctionComponent } from "react";
+import CommonModal from "../components/organisms/Modal";
+// import { createNewAccount } from "../services/auth";
```

Comment:
nit: remove unused import

Reply:
Thanks for the catch - but will keep this in there for now to remember to replace the import with the SignIn firebase service